src/cmd/manage_nsfs.js

```diff
@@ -596,6 +597,17 @@ async function delete_account(data) {
     write_stdout_response(ManageCLIResponse.AccountDeleted);
 }
 
+async function verify_delete_account(account_email) {
+    const show_secrets = true;
+    const bucket_config_files = await list_config_files(buckets_dir_path, show_secrets);
+    for (const bucket_config_file of bucket_config_files) {
+        if (bucket_config_file.bucket_owner === account_email) {
+            const detail_msg = `account email ${account_email}`;
```

Comment:
Example: by running `sudo node src/cmd/manage_nsfs account delete --name shira-50` you'll see:

>   "error": {
>     "code": "AccountDeleteForbiddenHasBuckets",
>     "message": "Cannot delete account that is owner of buckets. You must delete all buckets before deleting the account",
>     "detail": "account email shira-50@noobaa.com"
>   }

Which details would you like to add?
The price is that I would either pass more arguments to the function `verify_delete_account` or pass the data of the account, both of which I don't like.

Reply:
Well, when you are trying to delete an account, you know which one is it...
If we block the deletion of the account because there is a bucket associated with it, we should inform which bucket.
WDYT?

Reply:
I don't have this information now... I can this information but it has a price:
1. First, we will be back to the loop instead of `some` (I need to create the bucket list).
2. Second, it will damage the performance since now we will always iterate all the buckets (without stopping for the first one).
I guess if someone would like to list all his buckets he should use the list on the buckets and filter the account (I don't think it is implemented now).

Reply:
The message would be:
Example: by running `sudo node src/cmd/manage_nsfs account delete --name shira-50` you'll see:

> {
>   "error": {
>     "code": "AccountDeleteForbiddenHasBuckets",
>     "message": "Cannot delete account that is owner of buckets. You must delete all buckets before deleting the account",
>     "detail": "Account shira-50 has bucket bucket-shira-50"
>   }
> }